packages/core/injector/container-scanner.ts

```diff
@@ -35,7 +35,7 @@ export class ContainerScanner {
       : metatypeOrToken;
     const instanceWrapper = dependencies.get(name as string);
     if (!instanceWrapper) {
-      throw new UnknownElementException();
+      throw new UnknownElementException(name as string);
```

Comment:
```suggestion
      throw new UnknownElementException(name.toString());
```

`name` may be a symbol, therefore you need to call `toString()` so the token will get display as `Symbol(MY_TOKEN)`. #1551